br/pkg/storage/s3.go

```diff
@@ -110,9 +110,8 @@ func (rs *S3Storage) CopyFrom(ctx context.Context, e ExternalStorage, spec CopyS
 		Key:        aws.String(rs.options.Prefix + spec.To),
 	}
 
-	// NOTE: Maybe check whether the Go SDK will handle 200 OK errors.
-	// https://repost.aws/knowledge-center/s3-resolve-200-internalerror
```

Comment:
This was removed because I have checked the document and it reads:

> If you call this API operation directly, make sure to design your application to parse the content of the response and handle it appropriately. If you use AWS SDKs, SDKs handle this condition. The SDKs detect the embedded error and apply error handling per your configuration settings (including automatically retrying the request as appropriate). If the condition persists, the SDKs throw an exception (or, for the SDKs that don't use exceptions, they return an error).